CMakeLists.txt

```
@@ -111,4 +111,10 @@ if(NOT SKIP_GIT)
 endif()
 ### END  Git Version ###
 
+# Option to disable unity builds
+option(DISABLE_UNITY_BUILD "Disable unity build" OFF)
```

Negated names are weird, what do you think about

```suggestion
option(ENABLE_UNITY_BUILD "Disable unity build" ON)
```

instead?

---

Done. TYSM. Negated names are super hard to reason about indeed.